src/appComponents/twitterwall/singletweet.js

```diff
@@ -2,7 +2,7 @@
 angular
     .module('oe.twitterwall')
     .directive('tweet',tweet);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [A space is required after ','.](https://www.codacy.com/app/triplez/open-event-webapp/file/2412364420/issues/source?bid=3233857&fileBranchId=3235518#l4)